src/Aspire.Cli/Backchannel/AppHostBackchannel.cs

```diff
@@ -98,6 +98,11 @@ public async Task<string[]> GetPublishersAsync(CancellationToken cancellationTok
 
     public async IAsyncEnumerable<(string Id, string StatusText, bool IsComplete, bool IsError)> GetPublishingActivitiesAsync([EnumeratorCancellation]CancellationToken cancellationToken)
     {
+        if (_process!.HasExited)
```

Comment:
Isn't this racy?

Reply:
Yeah I need to take a closer look ... timing is very tight.